src/plcc/load_spec/validate_spec/validate_syntactic_spec/LL1/LL1Validator.py

```diff
@@ -0,0 +1,13 @@
+from .LL1Checker import check
+from .SpecGrammar import createSpecGrammar
+
+def checkLL1(syntactic_spec):
```

Comment:
For consistency, let's use `check_ll1` validator.

We use camelCasing for OO, and snake_casing otherwise.